gems/quilt_rails/lib/tasks/quilt_rails_tasks.rake

```diff
@@ -0,0 +1,70 @@
+require 'thor'
```

Comment:
For this you can define a generator that you can even test. Something like this: https://github.com/plataformatec/devise/blob/master/lib/generators/devise/install_generator.rb

Reply:
Cool, will redo as a generator 👍